src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -15,13 +15,18 @@
     :style="{ '--nesting-index': item.depth }"
     :aria-hidden="isRendered ? null : 'true'"
     :id="`container-${item.uid}`"
-    :aria-owns="ariaOwns"
-    :aria-level="item.depth"
-    :role="isParent ? 'group': 'treeitem'"
   >
     <div class="head-wrapper" :class="{ active: isActive, 'is-group': isGroupMarker }">
+      <span
+        v-if="isParent"
+        hidden
+        :id="buttonParentLabel"
+      >
+        {{ item.siblingsCount }} symbols to be {{ expanded ? 'collapsed' : 'expanded'}}
```

Comment:
This will close the childUIDs, not the siblings.

```suggestion
        {{ item.childUIDs.length }} symbols to be {{ expanded ? 'collapsed' : 'expanded'}}
```